portality/scripts/sage_update.py

```diff
@@ -0,0 +1,94 @@
+import csv
+
+from openpyxl import load_workbook
+from portality.models import Journal
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--out", help="output file path")
+    args = parser.parse_args()
+
+    if not args.out:
+        print("Please specify an output file path with the -o option")
+        parser.print_help()
+        exit()
+
+    with open(args.out, "w", encoding="utf-8") as f:
+        writer = csv.writer(f)
+
+        file = './resources/SAGE_jnls_in_DOAJ.edited.xlsx'
```

Comment:
probably best to keep the file outside our version control. upload it to /tmp/ or the home dir or similar on the server. There's no need to have it persist and bulk up our repo.

(since it's now been committed to this branch, it may be worth making a new branch & PR without it so it isn't in version control at all, or you can go through a 'purge file from repo' procedure)